src/dos/drive_overlay.cpp

```diff
@@ -209,11 +209,11 @@ class OverlayFile final : public localFile {
 			if (*data == 0) {
 				if (logoverlay) LOG_MSG("OPTIMISE: truncate on switch!!!!");
 			}
-			Bit32u a = GetTicks();
+			const auto a = logoverlay ? GetTicks() : 0;
 			bool r = create_copy();
-			if (GetTicks() - a > 2) {
-				if (logoverlay) LOG_MSG("OPTIMISE: switching took %d",GetTicks() - a);
-			}
+			const auto b = logoverlay ? GetTicksSince(a) : 0;
```

Comment:
Maybe `b` could be named `elapsed_ms`( if we're adding new code, might as well be self-documenting).

Reply:
yeah, it just looks a bit odd stuck in there with all the other single-letter variable names